src/CentOps.Api/Controllers/PublicParticipantController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AutoMapper;
+using CentOps.Api.Models;
+using CentOps.Api.Services.ModelStore.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CentOps.Api.Controllers
+{
+    [Route("public/participants")]
+    [ApiController]
+    public class PublicParticipantController : ControllerBase
+    {
+        private readonly IParticipantStore _store;
+        private readonly IMapper _mapper;
+
+        public PublicParticipantController(IParticipantStore store, IMapper mapper)
+        {
+            _store = store;
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult<IEnumerable<ParticipantResponseModel>>> Get()
+        {
+            var participants = await _store.GetAll().ConfigureAwait(false);
+
+            return Ok(_mapper.Map<IEnumerable<ParticipantResponseModel>>(participants));
+        }
+
+        [HttpGet("{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<IEnumerable<ParticipantResponseModel>>> Get(string id)
```

Comment:
Just a question, are we returning 1 single participant here? if so, we are returning it as a collection, perhaps this would be more suitable? please correct me if I have misunderstood
```suggestion
        public async Task<ActionResult<ParticipantResponseModel>> Get(string id)
```

Reply:
This one I have to admin I simply copied from ParticipantController. But it doesn't return a collection in either case, so I will adjust both of them to your recommendation, thanks!